azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/DependencyInjectionContainerHolder.kt

```diff
@@ -53,37 +46,4 @@ internal class DependencyInjectionContainerHolder(
 
         return@lazy CallCompositeInstanceManager.getCallComposite(instanceId).getDiContainer()
     }
-
```

Comment:
SetupViewModel be back to the view model to not to leak in the container, as it is statis